src/TMap.cpp

```diff
@@ -1759,7 +1763,7 @@ bool TMap::retrieveMapFileStats(QString profile, QString* latestFileName = nullp
     }
 
     if (otherProfileVersion >= 7) {
-        // hashTable
+        // hashToRoomID
         QHash<QString, int> _dummyQHashQStringInt;
```

Comment:
:fearful: Oh dear - I think I messed up here as these should be
```
    QMap<QString, int> _dummyQMapQStringInt;
    ifs >> _dummyQMapQStringInt;
```

Guess that either this isn't used that often or the two types are closer in binary format than they appear...